src/codecoverage/bot/tests/test__system.py

```diff
@@ -0,0 +1,29 @@
+# -*- coding: utf-8 -*-
```

Comment:
Nit: `test_system.py`

Reply:
I named it that way to be the first test to be executed: the process fails fast with `pytest -x` when you miss a dep on your local system